.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: "Test"
+on:
+  pull_request:
+  push:
+jobs:
+  tests:
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macos]
```

Comment:
Can you remove `macos`? It's not a target we support.

Reply:
Fixed!